jupyterhub/spawners.py

```diff
@@ -277,6 +277,9 @@ def get_pod_manifest(self):
             + "/"
             + options.get("project")
         )
+        git_host = urlparse(
+            os.environ.get("GITLAB_URL", "http://gitlab.renku.build"),
+        ).netloc
```

Comment:
Please add the `.path` here for the (admittedly unlikely) case that two gitlab instances are running under the same hostname, like `org.host/team-a/gitlab` and `org.host/team-b/gitlab`./

Reply:
@ableuler I would have to add the first portion of the path as a separate label. The reason for this is that k8s label values cannot contain `/`. Is this what you are referring? I.e. now there will be an extra label just called `git-path` and it would also sometimes be blank because sometimes gitlab is not deployed on `hostname.com/gitlab` but rather at `gitlab.something.com`.

Reply:
> The reason for this is that k8s label values cannot contain `/`

Ah, I did not think about this. 

> Is this what you are referring?

No, I was just thinking that since you add all the components of the repo URL as labels (which I like), these labels should also allow to uniquely identify a repo. With the path of the git host missing, this is not strictly the case. Admittedly, this would only matter when one RenkuLab instance can work with multiple GitLab instances AND two of those GitLab instances are running under different paths but share the same domain. Maybe just add the path and URL-encode it?

Reply:
Let me know if you agree with how I fixed this. I only took the first part of the `path`. So the following happens:
- if `GIT_URL = 'https://gitlab.com'`, then path is `""`
- if `GIT_URL = 'https://hostname.com/gitlab'`, then path is `"gitlab"`
- if `GIT_URL = 'https://hostname.com/team1/gitlab'`, then path is `"team1"`

Reply:
Hmm, tbh instead of only taking the first part of the path, I'd rather leave it out completely and go with your original version. I just thought that adding the full path to the GitLab instance would complete what you had wanted to achieve

> I essentially decided to add all the parts of the repository url as separate labels

but when suggesting that I didn't think about the `/`. So if URL-encoding the path is not an option for you, I'd rather not add it.

Reply:
Ok I will remove the path label then.